API/src/utils/Config/index.js

```diff
@@ -19,8 +19,8 @@ module.exports = {
     host: process.env.DATABASE_HOST || `localhost`,
     port: process.env.DATABASE_PORT || 5432,
     username: process.env.DATABASE_USERNAME || `postgres`,
-    password: process.env.DATABASE_PASSWORD || `paul`,
-    name: process.env.DATABASE_NAME || `risk`,
+    password: process.env.DATABASE_PASSWORD || `vikas003`,
+    name: process.env.DATABASE_NAME || `postgres`,
```

Comment:
```suggestion
    password: process.env.DATABASE_PASSWORD || `vikas003`,
    name: process.env.DATABASE_NAME || `postgres`,
```
You shouldn't put passwords in the code here.  This is my fault.  Use the config.json for this.